discord/ext/commands/bot.py

```diff
@@ -869,7 +920,7 @@ async def load_extension(self, name: str, *, package: Optional[str] = None) -> N
 
         await self._load_from_module_spec(spec, name)
 
-    async def unload_extension(self, name: str, *, package: Optional[str] = None) -> None:
+    async def unload_extension(self, name: str, /, *, package: Optional[str] = None) -> None:
```

Comment:
I think this one is fine as a dual keyword/positional.

Reply:
```suggestion
    async def unload_extension(self, name: str, *, package: Optional[str] = None) -> None:
```